subworkflows/local/annotate_vcfanno.nf

```diff
@@ -0,0 +1,25 @@
+//
+// Annotate with VCFanno
+//
+
+include { VCFANNO } from '../../modules/local/vcfanno/main'
+include { BCFTOOLS_ANNOTATE } from '../../modules/local/bcftools_annotate/main'
+
+workflow ANNOTATE_VCFANNO {
+    take:
+        vcf             // channel: [ val(meta), path(vcf), path(tbi) ]
+        resource_dir    // channel: path(resource_dir)
+
+    main:
+        ch_versions = Channel.empty()
+
+        VCFANNO (vcf, resource_dir)
+        ch_versions = ch_versions.mix(VCFANNO.out.versions)
+
+        BCFTOOLS_ANNOTATE ( VCFANNO.out.vcf )
+        ch_versions = ch_versions.mix(BCFTOOLS_ANNOTATE.out.versions)
```

Comment:
I would hold off with bcftools annotate since we're not feeding it any annotation files.